FirebasePerformance/Tests/Unit/FPRAppActivityTrackerTest.m

```diff
@@ -193,4 +205,88 @@ - (void)testApplicationStateManagement {
   XCTAssertEqual(appTracker.applicationState, FPRApplicationStateBackground);
 }
 
+- (void)testIsApplicationPrewarmedReturnsYesBecauseOfDoubleDispatch {
+  id mockAppTracker = OCMPartialMock([FPRAppActivityTracker sharedInstance]);
+  OCMStub([mockAppTracker isPrewarmAvailable]).andReturn(YES);
+  OCMStub([mockAppTracker isDoubleDispatchEnabled]).andReturn(YES);
+  OCMStub([mockAppTracker isActivePrewarmEnabled]).andReturn(NO);
+
+  [FPRAppActivityTracker load];
+  [[NSNotificationCenter defaultCenter]
+      postNotificationName:UIApplicationDidFinishLaunchingNotification
+                    object:[UIApplication sharedApplication]];
+
+  XCTAssertTrue([mockAppTracker isApplicationPreWarmed]);
+}
+
+- (void)testIsApplicationPrewarmedReturnsYesBecauseOfActivePrewarm {
```

Comment:
Just a thought on the testing function names - Consider shortening the names to a simple format.

go/unit-test-practices#naming

Typically the format should be "function/featture_scenario_expectedOutcome".